ext/debug/segger/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+
+zephyr_include_directories(${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
zephyr_include_directories(.)

would also work here, because the zephyr_* API will detect if a path is relative or absolute and then modify the path accordingly.